packages/components/src/components/horizontal-swipe/horizontal-swipe.tsx

```diff
@@ -1,7 +1,18 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { ReactNode } from 'react';
 import { Swipeable } from 'react-swipeable';
 
+type THorizontalSwipeProps = {
+    is_left_swipe: boolean;
+    is_left_swipe_disabled: boolean;
+    is_right_swipe: boolean;
+    is_right_swipe_disabled: boolean;
+    left_hidden_component: ReactNode;
+    left_hidden_component_width?: string;
```

Comment:
```suggestion
    is_left_swipe: boolean;
    is_left_swipe_disabled?: boolean;
    is_right_swipe?: boolean;
    is_right_swipe_disabled?: boolean;
    left_hidden_component?: ReactNode;
    left_hidden_component_width?: string;
```
some more props are optional 🙂 @iman-fs

Reply:
thanks for that